tests/TestCase/ORM/Behavior/TranslateBehaviorTest.php

```diff
@@ -894,7 +894,7 @@ public function testUpdateSingleLocale()
         $this->assertSame('New translated article', $article->get('title'));
         $this->assertSame('Content #1', $article->get('body'));
 
-        $table->setLocale(false);
+        $table->setLocale(null);
```

Comment:
Did changing the calls trigger argument validation somewhere?

Reply:
Yup, the argument for `TranslateStrategyInterface::setLocale()` is typehinted as `?string`. I don't know why TypeError was not thrown earlier.